server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/redirect/MailModifier.java

```diff
@@ -120,7 +120,7 @@ public void setTo(List<MailAddress> mailAddresses) throws MessagingException {
             InternetAddress[] internetAddresses = MailAddressUtils.toInternetAddressArray(mailAddresses);
             mail.getMessage().setRecipients(Message.RecipientType.TO, internetAddresses);
             if (mailet.getInitParameters().isDebug()) {
-                LOGGER.debug("apparentlyTo set to: " + internetAddresses);
+                LOGGER.debug("apparentlyTo set to: {}", (Object) internetAddresses);
```

Comment:
Why this cast ?

Reply:
The cast was to avoid the varargs debug() overload from being called, because `internetAddresses` is an array.

I now think Arrays.asList() should be used here, and I just pushed a commit to make this change.